utils/src/main/java/com/cloud/utils/PropertiesUtil.java

```diff
@@ -34,6 +34,10 @@
 public class PropertiesUtil {
     private static final Logger s_logger = Logger.getLogger(PropertiesUtil.class);
 
+    public static String getDefaultApiCommandsFileName() {
+        return "commands.properties";
+    }
```

Comment:
PropertiesUtil deals with commands.properties file related processing, so I've put it here. Also, utils is a common packages used by several other modules, putting this in an authenticator (server or plugin package) won't make it generally consumable across several modules if needed.